src/main/resources/dbreplica/migration/V1.0.32__task_assignments_plsql.sql

```diff
@@ -0,0 +1,66 @@
+--
+-- RWA-2382 Trigger on reportable_task to create and update record in task_assignments for every assignee of task.
+--
+CREATE OR REPLACE FUNCTION cft_task_db.add_task_assignments(l_task_id text)
+ RETURNS bigint
+ LANGUAGE plpgsql
+AS $function$
+
+declare
+l_update_id     bigint;
+l_assignee      text;
+l_update_action text;
+l_updated       TIMESTAMP;
+l_state         text;
+l_jurisdiction  text;
+l_location      text;
+l_role_category text;
+l_task_name     text;
+l_new_task      boolean;
+l_valid_history boolean DEFAULT false;
```

Comment:
Is this provided from elsewhere or is this always false?

Reply:
Hello Ravi, Default is false but I am setting it to true in line 45.